Wikipedia/Code/TalkPageHeaderView.swift

```diff
@@ -335,7 +335,21 @@ final class TalkPageHeaderView: SetupView {
         
         let theme = viewModel.theme
         
-        coffeeRollLabel.attributedText = coffeeRollText.byAttributingHTML(with: .callout, boldWeight: .semibold, matching: traitCollection, color: theme.colors.primaryText, linkColor: theme.colors.link, handlingLists: true, handlingSuperSubscripts: true, tagMapping: ["a": "b"])
+        var attributedText: NSMutableAttributedString = coffeeRollText.byAttributingHTML(with: .callout, boldWeight: .semibold, matching: traitCollection, color: theme.colors.primaryText, linkColor: theme.colors.link, handlingLists: true, handlingSuperSubscripts: true, tagMapping: ["a": "b"])
+        if attributedText.string.first == "\n" {
+            attributedText = removeInitialCharactersFrom(&attributedText)
+        }
+        coffeeRollLabel.attributedText = attributedText
+    }
+    
+    // Coffee roll attributed strigs were sometimes created with extra new line characters (\n) as the initial characters, so we remove them to properly render the text
+    func removeInitialCharactersFrom(_ attributedString: inout NSMutableAttributedString) -> NSMutableAttributedString {
```

Comment:
Can you change this into an NSAttributedString extension instead? That way it will be easier to use in the talk page cells. It would be nice if the callsite could look something like:

```
let coffeeRollAttributedText = coffeeRollText.byAttributingHTML(with: .callout, boldWeight: .semibold, matching: traitCollection, color: theme.colors.primaryText, linkColor: theme.colors.link, handlingLists: true, handlingSuperSubscripts: true, tagMapping: ["a": "b"])
cofeeRollLabel.attributedText = coffeeRollAttributedText.removingInitialNewlineCharacters()
```